### PR TITLE
Require --containers and ensure at least one test

### LINKED ISSUE #2: Output usage (or _anything_) if no/wrong arguments are used
Seems --containers is required, as is at least one of --status --memory and --restarts

Should be reflected if none/wrong is supplied.

Also, as such is the case, the help output shouldn't put --containers in square brackets as that indicates it's optional.

### PR DESCRIPTION
**Pull request by KristianLyng**
Fixes #2

Granted, this is two things in one.

The --containers seems fairly obvious to me.

"checks" could probably be documented slightly better, but this will at
least give the user a hint.

### CODE REVIEW

File: check_docker.py [hunk 1]
```diff
@@ -224,9 +224,10 @@ def process_args(args):
 
     # Container name
     parser.add_argument('--containers',
+                        required=True,
```
Comment (timdaman) [inline check_docker.py]: I get why you are requiring '--containers' but I think you are missing a use case. I envision want to install this on a server with lots of container workloads(maybe a utility box) which may a lot of different containers or ones that change frequently. By defaulting to 'all' we can monitor whatever workloads there are without having to right a check for each.  

As a result of this I would rather not merge this improvement.

File: check_docker.py [hunk 2]
```diff
@@ -288,6 +289,7 @@ def print_results():
 
     #############################################################################################
     args = process_args(argv[1:])
+    checks = 0
```
Comment (timdaman) [inline check_docker.py]: Testing for 'checks' is a great idea. We should really add a unit test that, I can take care of that if you don't want to. I am happy top merge this function anytime.

File: check_docker.py (hunk 1 above)
Comment (KristianLyng) [inline check_docker.py]: Hmm, I would argue it makes more sense for "all" to be explicit, but I honestly didn't realize it used it by default, largely due to the lack of a default test at all. I'll remove the requirement.

**Comment by KristianLyng** [comment]
I added a commit removing the "Required = true" bit, but left nargs = "+" instead of "*", and checks.

**Pull request merged**